eventdiscovery-sdk/src/main/res/layout/schedjoules_fragment_event_details.xml

```diff
@@ -24,14 +24,78 @@
             <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:layout_marginTop="8dp"
                     android:orientation="vertical">
 
-                <LinearLayout
-                        android:id="@+id/schedjoules_event_detail_vertical_items"
-                        android:orientation="vertical"
+                <include
+                        layout="@layout/schedjoules_view_event_details_venue"
+                        android:id="@+id/schedjoules_event_details_venue_name"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:layout_marginTop="12dp"/>
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginRight="16dp"
+                        android:layout_marginTop="8dp"
+                        android:layout_marginBottom="8dp"/>
+
+                <include
+                        layout="@layout/schedjoules_view_event_details_datetime"
+                        android:id="@+id/schedjoules_event_details_datetime"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginRight="16dp"
+                        android:layout_marginTop="8dp"
+                        android:layout_marginBottom="8dp"/>
+
+                <include
+                        layout="@layout/schedjoules_view_event_details_description"
+                        android:id="@+id/schedjoules_event_details_description"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginRight="16dp"
+                        android:layout_marginTop="8dp"
+                        android:layout_marginBottom="8dp"/>
+
+                <include
+                        layout="@layout/schedjoules_view_event_details_action"
```

Comment:
a lot of repetition here. Please create a separate style. Also watch out for the box model. Often it's better to give the element the same margin on all sides (e.g. 8dp) and have the missing 8dp added by a padding on the container element.